pytest-embedded-serial-esp/pytest_embedded_serial_esp/serial.py

```diff
@@ -9,11 +10,26 @@
 from esptool import __version__ as ESPTOOL_VERSION
 from esptool import detect_chip
 from esptool.targets import CHIP_LIST as ESPTOOL_CHIPS
-from pytest_embedded.log import MessageQueue
+from pexpect import TIMEOUT
+from pytest_embedded.log import MessageQueue, _PexpectProcess, live_print_call
 from pytest_embedded.utils import Meta
 from pytest_embedded_serial.dut import Serial
 
 
+def is_port_mac_verified(pexpect_proc: _PexpectProcess, port: str, port_mac: str, msg_queue) -> bool:
```

Comment:
Then I have to create the instance attributes for all of these parameters since at the moment when this function called there is no any of these attributes available.

Reply:
I agree. how about making this function starts with `_`? in order to not be generated in the API reference.